From eb1cf0eae21f4be89534323f126e88a8a0ce047b Mon Sep 17 00:00:00 2001
From: Pier Luigi Fiorini <pierluigi.fiorini@gmail.com>
Date: Tue, 30 Apr 2013 20:20:25 +0200
Subject: [PATCH] Add desktop file

We need a desktop file for Maui otherwise users won't be able to see
and run QtCreator.
---
 src/app/app.pro           |    4 ++++
 src/app/qtcreator.desktop |   10 ++++++++++
 2 files changed, 14 insertions(+)
 create mode 100644 src/app/qtcreator.desktop

diff --git a/src/app/app.pro b/src/app/app.pro
index 1242209..b05f159 100644
--- a/src/app/app.pro
+++ b/src/app/app.pro
@@ -41,3 +41,7 @@ OTHER_FILES += qtcreator.rc \
 QMAKE_SUBSTITUTES += $$PWD/app_version.h.in
 
 CONFIG += no_batch
+
+desktop.files = qtcreator.desktop
+desktop.path = $$QTC_PREFIX/share/applications
+INSTALLS += desktop
diff --git a/src/app/qtcreator.desktop b/src/app/qtcreator.desktop
new file mode 100644
index 0000000..ce95b78
--- /dev/null
+++ b/src/app/qtcreator.desktop
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Type=Application
+Exec=qtcreator %F
+Name=Qt Creator
+GenericName=C++ IDE for developing Qt applications
+X-KDE-StartupNotify=true
+Icon=qtcreator
+Terminal=false
+Categories=Development;IDE;Qt;
+MimeType=text/x-c++src;text/x-c++hdr;text/x-xsrc;application/x-designer;application/vnd.nokia.qt.qmakeprofile;application/vnd.nokia.xml.qt.resource;
-- 
1.7.10.4

